#! /usr/bin/env python

PACKAGE='experiment1'
import roslib
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
#       Name         Type      Level     Description    Default  Min   Max
gen.add("delta",    double_t,    0,    "The message.",    0.5,   0.2,    1 )
gen.add("delta_s",  double_t,    0,    "Third number.",   0.2,    0,    0.5)
gen.add("vel_max",  double_t,    0,    "First number.",   0.3,    0,     1 ) 
gen.add("vel_min",  double_t,    0,    "First number.",   0.1,    0,    0.5) 
gen.add("ang_max",  double_t,    0,    "Second number.",   1,     0,     3 )


exit(gen.generate(PACKAGE, "experiment1", "composite"))